addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -32,6 +35,21 @@ var SmsWidget = FieldTextEmojis.extend({
         }
     },
 
+    /**
+     * Open a Model Field Selector in order to select fields
+     * and create a dynamic placeholder string with or without
+     * a default text value.
+     *
+     * @public
+     * @param {String} baseModel
+     * @param {Array} chain
+     *
+     */
+    openDynamicPlaceholder: async function (baseModel, chain = []) {
+        const modelSelector = await this._openNewModelSelector(baseModel, chain);
+        modelSelector.$el.css('margin-top', 4);
```

Comment:
I don't really dare, this widget is used in many other place in Odoo, So I took the easy route.
I might be improved with CSS indeed. But I don't realy know how long it would take to be sure it doesn't mess up at other places.
Except if I flag the widget with a custom class, but it seems a bit overkill IMO.

WYT?